01-js-intro-and-jquery/keypoints/1_11/resources/script/script.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * Created by MaximilianoDaniel on 11/09/2015.
+ */
+
+var onPageReady = function () {
+    var processRequest;
+    var showError;
+    var showComplete;
+    var listLenght;
+    var listAlbums;
+    var showAlbum;
+
+    showAlbum = function(album){
+
+    };
+    showError = function(xmlHttpRequest,errorText,objectException) {
+        window.alert('error');
+    };
+
+    showComplete = function(xmlHttpRequest,result) {
+        window.alert('complete');
+    };
+
+    processRequest = function(result) {
```

Comment:
most of the times we call our function with more detailed names and with some kind of conventions like:

For request or event when the event happends we call the function onEventSuccess or onEventHappend or onButtonClick as example. It's ok anyway, just a comment to have in mind